src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/jobs/NotifyAttentionNeededLicencesService.kt

```diff
@@ -62,4 +60,22 @@ class NotifyAttentionNeededLicencesService(
     )
     log.info("Job notifyAttentionNeededLicences emailed ${attentionNeededLicences.size} licences")
   }
+
+  fun List<NotifyAttentionNeededLicence>.toCsv(): String {
+    val csv = StringBuilder()
+    csv.append("Noms ID,Prison Name,Noms Legal Status,ARD,CRD,Licence Start Date\r\n")
+    this.forEach {
+      csv.append("${it.nomsId},${it.prisonName},${it.legalStatus},${it.actualReleaseDate},${it.conditionalReleaseDate},${it.licenceStartDate}\r\n")
+    }
+    return csv.toString()
+  }
+
+  data class NotifyAttentionNeededLicence(
+    val nomsId: String?,
+    var prisonName: String? = null,
+    var legalStatus: String? = null,
+    val conditionalReleaseDate: LocalDate?,
+    val actualReleaseDate: LocalDate?,
+    val licenceStartDate: LocalDate?,
+  )
```

Comment:
This is all moving packages around apart from this bit where I've inlined a class and associated toCsv library. 
Tidied up the associated test a little too